packages/node_modules/pouchdb-ajax/src/request-browser.js

```diff
@@ -213,12 +213,14 @@ function xhRequest(options, callback) {
       if (timedout) {
         err = new Error('ETIMEDOUT');
         err.code = 'ETIMEDOUT';
-      } else if (typeof xhr.response === 'string') {
+      } else if (typeof xhr.response === 'string' && xhr.response !== '') {
         try {
           err = JSON.parse(xhr.response);
         } catch (e) {}
       }
-      err.status = xhr.status;
+      if (typeof err === 'object') {
```

Comment:
Since this isnt executed as part of the test suite its now failing coverage, usually we would require a test that covers this however since this code is going to be removed on the next release and the code change is fairly safe I think its ok to add an istanbul ignore in there

Reply:
Ok, thanks, I will delete these changes and recreate pr.